module/tools/RoboCupConfiguration/src/RoboCupConfiguration.cpp

```diff
@@ -121,6 +121,14 @@ namespace module::tools {
                     system("systemctl stop robocup");
                     log_message = "RoboCup service stopped!";
                     break;
+                case 'E':  // enables wifi services
+                    system("systemctl enable wpa_supplicant");
```

Comment:
`enable` enables a service at startup. Use `enable --now` to enable on startup and start immediately. Same with disable.